### PR TITLE
geeksoftwareGmbH.PDF24Creator version 11.11.0

### PR DESCRIPTION
**Pull request by borque**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

### CODE REVIEW

File: manifests/g/geeksoftwareGmbH/PDF24Creator/11.11.0/geeksoftwareGmbH.PDF24Creator.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: geeksoftwareGmbH.PDF24Creator
+PackageVersion: 11.11.0
+InstallerLocale: en-US
```
Comment (mdanish-kh) [inline manifests/g/geeksoftwareGmbH/PDF24Creator/11.11.0/geeksoftwareGmbH.PDF24Creator.installer.yaml:6]:
```suggestion
InstallerLocale: en-US
Scope: machine
```
Package install machine-wide

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by borque** [comment]
Hello @mdanish-kh !
I wasn't aware, that "scope" is meanwhile necessary, as I have ommited it in previous versions of this manifest. But of  course you are right, it installs machine wide!

**Comment by mdanish-kh** [comment]
It's not necessary, but it would enable the use of a command like `winget install geeksoftwareGmbH.PDF24Creator --scope machine` if someone is using --scope parameter in their automation / scripts.

**Comment by borque** [comment]
@mdanish-kh : With this new addition the PR fails validation?!

**Review by mdanish-kh — APPROVED** [review]

**Comment by borque** [comment]
Funny, now it's running again!
Thanks for your explanation concerning "scope"!

**Pull request merged**